pkg/tgsync/tgsync.go

```diff
@@ -125,5 +124,10 @@ func runTgSyncTest(runenv *runtime.RunEnv, initCtx *run.InitContext, ctx context
 		return err
 	}
 
+	filePath, err := runenv.CreateRandomFile("./", 2*1024) // 2MB
+	if err != nil {
+		return err
+	}
+	fmt.Println("Output file created on location: " + filePath)
```

Comment:
What is the use of this random file?

Reply:
You were right, it is useless.. also the container gets deleted after successful plan execution so it can't be accessed anyway